ASPNetCoreMastersTodoList.Api/Controllers/ItemsController.cs

```diff
@@ -0,0 +1,59 @@
+﻿using ASPNetCoreMastersTodoList.Api.BindingModels;
+using Microsoft.AspNetCore.Mvc;
+using Services.DTO;
+using Services.Interfaces;
+using System.Collections.Generic;
+
+namespace ASPNetCoreMastersTodoList.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ItemsController : ControllerBase
+    {
+        private readonly IItemService _itemService;
+        public ItemsController(IItemService itemService)
+        {
+            _itemService = itemService;
+        }
+
+        [HttpGet()]
+        public IActionResult GetAll()
+        {
+            var items = _itemService.GetAll();
+            return Ok(items);
+        }
+
+        [HttpGet("{itemId}")]
+        public IActionResult Get(int itemId)
+        {
+            return Ok(_itemService.GetById(itemId));
+        }
+
+        [HttpGet("filter")]
```

Comment:
Hi sir, can you please change the route to "filterBy" as indicated on the homework? Very minor but it will be easier for us proctors to check.

Reply:
updated